core/src/main/java/org/elasticsearch/action/admin/cluster/allocation/ClusterAllocationExplanation.java

```diff
@@ -76,18 +84,26 @@ public ClusterAllocationExplanation(StreamInput in) throws IOException {
         }
         this.nodeWeights = ntw;
         remainingDelayNanos = in.readVLong();
+        size = in.readVInt();
```

Comment:
It looks like readList doesn't but writeList does. There isn't a Streamable version. Oh well.